packages/11ty/_includes/web-components/image-sequence/index.js

```diff
@@ -246,6 +285,12 @@ class ImageSequence extends LitElement {
     }
   }
 
+  /**
+   * @function hideOverlays
+   * 
+   * TODO: Should be removed in favor of dynamic markup (may already be)
+   * 
+   **/ 
```

Comment:
Use [JSDoc todo tag](https://jsdoc.app/tags-todo)
```suggestion
  /**
   * @function hideOverlays 
   * @todo removed in favor of dynamic markup (may already be)
   */ 
```

Reply:
Incorporated (removed hideOverlays function and call in favor of relying on didInteract sentinel)